compiler/ir/ir.tree/tree-generator/src/org/jetbrains/kotlin/ir/generator/print/Common.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.ir.generator.print
+
+import com.squareup.kotlinpoet.FileSpec
+import com.squareup.kotlinpoet.TypeSpec
+import org.jetbrains.kotlin.ir.generator.util.GeneratedFile
+import java.io.File
+
+private val COPYRIGHT by lazy { File("license/COPYRIGHT_HEADER.txt").readText() }
+private val GENERATED_MESSAGE = """
+     //This file was generated automatically
```

Comment:
I've decided not to change this because it would regenerate all of FIR tree too. Having different generated messages seems weird though. Let's fix it afterwards?

Reply:
ok